cmd/main.go

```diff
@@ -210,6 +214,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	kubeClient := mgr.GetClient()
+	if err := mgr.Add(utils.NewRecycleCronJob(kubeClient, recycleDuration, "0 0 * * *", groupByLabel)); err != nil {
```

Comment:
If the groupByLabel is empty, we do not need to create the recycling job?

Reply:
Great catch!